spark/src/main/scala/geotrellis/spark/regrid/Regrid.scala

```diff
@@ -118,11 +118,12 @@ object Regrid {
 
               val xSpan: Interval[Long] = oldXrange intersect newXrange
               val ySpan: Interval[Long] = oldYrange intersect newYrange
+              val forceCrop = newW < 0.6*oldW || newH < 0.6*oldH
               newKey ->
                 (oldTile.crop((xSpan.start - oldXstart).toInt,
                               (ySpan.start - oldYstart).toInt,
                               (xSpan.end - oldXstart).toInt,
-                              (ySpan.end - oldYstart).toInt),
+                              (ySpan.end - oldYstart).toInt),Crop.Options(force=forceCrop)),
                  ((xSpan.start - newXrange.start).toInt, (ySpan.start - newYrange.start).toInt)
                 )
```

Comment:
It does not allow me to add a code change suggestion, but should be smth like:

```scala
              val forceCrop = newW < 0.6 * oldW || newH < 0.6 * oldH
              newKey ->
                (
                  oldTile.crop(
                    (xSpan.start - oldXstart).toInt,
                    (ySpan.start - oldYstart).toInt,
                    (xSpan.end - oldXstart).toInt,
                    (ySpan.end - oldYstart).toInt,
                    Crop.Options(force = forceCrop)
                  ),
                  ((xSpan.start - newXrange.start).toInt, (ySpan.start - newYrange.start).toInt)
                )
```

Crop options is an argument to the crop function.

Reply:
I made up a https://github.com/locationtech/geotrellis/pull/3518 to try out the CI, but please feel free to commit changes as a part of this PR as well. 

U can also let me pushing into your PR to speed up the progress! 

![image](https://github.com/locationtech/geotrellis/assets/4929546/085102f9-3738-4e59-8052-e95b12eac719)